pyprof/nvtx/nvmarker.py

```diff
@@ -74,7 +71,7 @@ def stop_graph():
     """
     global wrappers_enabled
     wrappers_enabled = False
-    dprint("Stopping graph tracker wrappers enabled {}".format(wrappers_enabled))
+    dprint(f"Stopping graph tracker wrappers enabled {wrappers_enabled}")
     return
```

Comment:
I am not sure, but these functions appear to be dlprof specific (?) and so they should be part of that class.